docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.8'
+
+services:
+  web:
+    build: .
```

Comment:
Do we need to write a Dockerfile for building the container for the 'web' service itself?

Reply:
@Aiderbot Do we need to write a Dockerfile for building the container for the 'web' service itself?